go-delegator/go.mod

```
@@ -0,0 +1,159 @@
+module bisonai.com/orakl/go-delegator
```

I think we could call it `delegator` already know. It has no overlap with js delegator namespace.

```suggestion
module bisonai.com/orakl/delegator
```